docs/modules/qbi-helper/interest-query.md

```diff
@@ -0,0 +1,53 @@
+<%(TOC)%>
+
+# How to construct an InterestQuery
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Introduction to Query-based interest]({{urlRoot}}/modules/qbi-helper/intro-to-qbi)
+  * [How to use InterestTemplate]({{urlRoot}}/modules/qbi-helper/interest-template)
+")%>
+
+An [`InterestQuery`]({{urlRoot}}/api/query-based-interest/interest-query) object contains the constraint of a query and the components that the query will return from entities that match the given constraint.
+
+> In the future, the Runtime will also provide functionality to limit the frequency at which a query returns results.
```

Comment:
Is this disclosed anywhere else? If not, lets not put this here

Reply:
It's mentioned in the upcoming features section of the current QBI page: https://docs.improbable.io/reference/13.6/shared/reference/query-based-interest#upcoming-features